pimatic-led-light.coffee

```diff
@@ -17,9 +17,9 @@ module.exports = (env) ->
     init: (app, @framework, @config) =>
       deviceConfigDef = require("./led-light-schema")
 
-      @framework.deviceManager.registerDeviceClass "LedLight",
+      @framework.deviceManager.registerDeviceClass "IWYLed",
```

Comment:
Yea you got it right ;)

So only The plugin itself is seperate. All of The action providers ect. And The gui is shared